tests/connect/compression_error-001.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+MongoDB\Driver\Manager: Connecting with unsupported compressor
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
```

Comment:
`CLEANUP(STANDALONE);` shouldn't be needed here, since we neither do IO nor assert existing data.